angular/src/app/modules/delivery-management/delivery/request-resource-tab/form-resource-requestCV/form-resource-requestCV.component.ts

```
@@ -134,22 +134,20 @@ export class ResourceRequestCVComponent extends AppComponentBase implements OnIn
-          this.dialogRef.close(this.resourceRequestCV);
-       },()=> this.isLoading = false)
-   }else{
-     let createResourceRequestCV = this.formatInterviewDateToVN(resourceCv);
-    this.resourceRequestService.updateResourceRequestCV(createResourceRequestCV).pipe(catchError(this.resourceRequestService.handleError)).subscribe((res)=>{
+  if (this.input.command == "create") {
+    resourceCv.id = 0
+    let createResourceRequestCV = this.formatInterviewDateToVN(resourceCv);
+    this.resourceRequestService.addCV(createResourceRequestCV).pipe(catchError(this.resourceRequestService.handleError)).subscribe((res) => {
+    abp.notify.success("Create SuccessFully");
+    this.dialogRef.close(createResourceRequestCV);
+    }, () => this.isLoading = false)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)